src/Microsoft.AspNet.Diagnostics.Elm/Views/LogPage.cs

```diff
@@ -306,6 +306,8 @@ public LogPage()
         #pragma warning disable 1998
         public override async Task ExecuteAsync()
         {
+            Response.ContentType = "text/html; charset=utf-8";
+            Response.ContentLength = null; // Clear any prior Content-Length
```

Comment:
There shouldn't be a prior content-length for either of these pages, did you see one?